mirrord/layer/tests/outgoing.rs

```diff
@@ -137,3 +145,26 @@ async fn outgoing_tcp(dylib_path: &PathBuf) {
 
     test_process.wait_assert_success().await;
 }
+
+#[rstest]
+#[tokio::test]
+#[timeout(Duration::from_secs(10))]
+async fn outgoing_tcp(
+    #[values(None, Some("outgoing_filter.json"))] with_config: Option<&str>,
+    dylib_path: &PathBuf,
+    config_dir: &PathBuf,
+) {
+    outgoing_tcp_logic(with_config, dylib_path, config_dir).await;
+}
+
+#[rstest]
+#[tokio::test]
+#[timeout(Duration::from_secs(10))]
+#[should_panic]
+async fn outgoing_tcp_local_broken(
```

Comment:
why "broken"?

Reply:
Oh, I see. Sure, you could listen for a connection in the test's code. But I'm not saying you have to, the test seems fine to me.
Either way please add documentation explaining what this test verifies and how.

Reply:
Added docs, are they clear enough though?